### PR TITLE
Update the ingress controller task.

### PR DESCRIPTION
**Pull request by myidpt**
Added the secure ingress verification steps.
Updated the response code (403 -> 404) in the verification.

### CODE REVIEW

**Comment by myidpt** [comment]
@wattli @ldemailly Can you please take a look? Thanks.

File: _docs/tasks/traffic-management/ingress.md [hunk 1]
```diff
@@ -205,21 +198,85 @@ The following are the known limitations of Istio ingress:
...
    ```
 
-   Set the INGRESS_HOST to point to the ip address and the
-   port number of the ingress service as shown earlier.
+   > Note: Because SNI is not yet supported, Envoy currently only allows a single TLS secret in the ingress.
+   > That means the secretName field in ingress resource is not used.
 
-   > Note: Envoy currently only allows a single TLS secret in the ingress
-   > since SNI is not yet supported. That means that the secret name field
-   > in ingress resource is not used, and the secret must be called
```
Comment (ldemailly) [inline _docs/tasks/traffic-management/ingress.md]: i believe you need this part, the cert must be called istio-ingress-certs

Comment (myidpt) [reply _docs/tasks/traffic-management/ingress.md]: OK. I can add it back to make it more explicit.

Comment (myidpt) [reply _docs/tasks/traffic-management/ingress.md]: Added to Line 170. Thanks.

**Pull request merged**